tests/pfc/files/configs/pfc.py

```diff
@@ -0,0 +1,303 @@
+import time
+import pytest
+import sys
+
+from tests.common.reboot import logger
+from tests.common.helpers.assertions import pytest_assert
+from tests.common.ixia.ixia_helpers import IxiaFanoutManager, get_location
+
+from tests.common.ixia.common_helpers import get_vlan_subnet, \
+    get_addrs_in_subnet
+
+###############################################################################
+# Imports for Tgen and IxNetwork abstract class
+###############################################################################
+
+from abstract_open_traffic_generator.port import Port
+from abstract_open_traffic_generator.result import PortRequest
+from abstract_open_traffic_generator.config import Options
+from abstract_open_traffic_generator.config import Config
+
+from abstract_open_traffic_generator.layer1 import\
+    Layer1, OneHundredGbe, FlowControl, Ieee8021qbb
+
+from abstract_open_traffic_generator.device import\
+    DeviceGroup, Device, Ethernet, Vlan, Ipv4, Pattern
+
+from abstract_open_traffic_generator.flow import\
+    DeviceEndpoint, Endpoint, Flow, Header, Size, Rate,\
+    Duration, Fixed, PortEndpoint, PfcPause, Counter, Random
+
+from abstract_open_traffic_generator.flow_ipv4 import\
+    Priority, Dscp
+
+from abstract_open_traffic_generator.flow import Pattern as FieldPattern
+from abstract_open_traffic_generator.flow import Ipv4 as Ipv4Header
+from abstract_open_traffic_generator.flow import Ethernet as EthernetHeader
+from abstract_open_traffic_generator.port import Options as PortOptions
+
+@pytest.fixture
+def start_delay():
+    start_delay = 1
+    return start_delay
+
+
+@pytest.fixture
```

Comment:
traffic_duration should be defined in the test script as different test cases may have different durations